bin/run-integrationtests

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -o errexit
+
+trap teardown EXIT
+
+function setup() {
+    mv resources/config.edn resources/config.edn.tmp | true
```

Comment:
This throws `mv: cannot stat 'resources/config.edn': No such file or directory`

Reply:
will redirect to /dev/null